src/darwin/Framework/CHIPTests/MTRCertificateTests.m

```diff
@@ -162,25 +162,25 @@ - (void)testGenerateOperationalCertErrorCases
     __auto_type * operationalKeys = [[MTRTestKeys alloc] init];
     XCTAssertNotNil(operationalKeys);
 
-    __auto_type * longCats = [[NSMutableArray alloc] initWithCapacity:4];
+    __auto_type * longCats = [[NSMutableSet alloc] initWithCapacity:4];
     [longCats addObject:@0x00010001];
     [longCats addObject:@0x00020001];
     [longCats addObject:@0x00030001];
     [longCats addObject:@0x00040001];
 
-    __auto_type * catsWithSameIdentifier = [[NSMutableArray alloc] initWithCapacity:3];
+    __auto_type * catsWithSameIdentifier = [[NSMutableSet alloc] initWithCapacity:3];
     // High bits are identifier, low bits are version.
     [catsWithSameIdentifier addObject:@0x00010001];
     [catsWithSameIdentifier addObject:@0x00020001];
     [catsWithSameIdentifier addObject:@0x00010002];
 
-    __auto_type * catsWithDuplicatedCAT = [[NSMutableArray alloc] initWithCapacity:3];
```

Comment:
Looks like this scenario no longer makes sense due to using a set

Reply:
Yeah I think so, wouldn't that scenario even fail now since it's expecting an error due to the duplicate?

Reply:
Per Slack discussion: yes, and done.